autogpt_platform/backend/backend/executor/scheduler.py

```diff
@@ -303,6 +314,7 @@ def run_service(self):
                 Jobstores.WEEKLY_NOTIFICATIONS.value: MemoryJobStore(),
             },
             logger=apscheduler_logger,
+            timezone=ZoneInfo("UTC"),
```

Comment:
This is explicitly setting the scheduler to utc zone rather than whatever it feels like